vscode/src/rubyLsp.ts

```diff
@@ -98,6 +98,20 @@ export class RubyLsp {
       await this.activateWorkspace(firstWorkspace, true);
     }
 
+    // If the user has the editor already opened on a Ruby file and that file does not belong to the first workspace,
+    // eagerly activate the workspace for that file too
+    const activeDocument = vscode.window.activeTextEditor?.document;
+
+    if (activeDocument && activeDocument.languageId === "ruby") {
```

Comment:
Does it make sense to include `erb` as well?

Reply:
It does make sense and now that you mentioned it, I realized we needed a better way of sharing which languages are supported for multiple places in the extension.

The latest commit creates a `SUPPORT_LANGUAGE_IDS` constant, that we then use to register for lazy activation, for the LSP client and for the language status item.